cosmwasm/contracts/pyth/src/state.rs

```diff
@@ -39,20 +36,29 @@ pub const VALID_TIME_PERIOD: Duration = Duration::from_secs(3 * 60);
 
 #[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, Eq, Hash, JsonSchema)]
 pub struct PythDataSource {
-    pub emitter: Binary,
+    pub emitter:            Binary,
     pub pyth_emitter_chain: u16,
 }
 
 // Guardian set information
-#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct ConfigInfo {
-    pub owner: HumanAddr,
-    pub wormhole_contract:  HumanAddr,
-    pub data_sources: HashSet<PythDataSource>,
+    pub owner:             Addr,
+    pub wormhole_contract: Addr,
+    pub data_sources:      HashSet<PythDataSource>,
 }
 
+impl Default for ConfigInfo {
```

Comment:
If It’s not too late to change this API, I think we would be better using `Option<Addr>` rather than using unchecked empty strings as nullify values, we can also drop the Default impl then.